spec/support/barcode_helper.rb

```diff
@@ -1,3 +1,22 @@
+module Barcode
+  def self.boop_barcode(value, barcode_field = nil)
```

Comment:
This is an extremely minor non-blocker favor -- could you rename this method to just `boop`? :pray: :pray: :pray: 

Otherwise this is perfect :100:

Reply:
For sure, not a problem at all!